static/scss/answers/interactive-map/VerticalFullPageMap.scss

```diff
@@ -1,3 +1,5 @@
+$mobile-break-point-max: sm !default;
```

Comment:
Hm, this makes the SCSS variable global to any file that imports this file (e.g. `_default.scss`) instead of scoped to the VerticalFullPageMap. How do we feel about removing the uses of `$mobile-break-point-max` and `$desktop-break-point-min` in this file and instead using some variable in `answers-variables`? Maybe we could make a new one for vertical-full-page-maps?

Reply:
I'm not sure about exposing this an answers-variables, because we have the mobile breakpoint hardcoded if a few places, and we didn't really design the map to support different breakpoints. Due to a refactor, this change is no longer necessary, however the ability to change the breakpoint might make sense in a future enhancement